envs/conda_build_config.yaml

```diff
@@ -282,6 +287,8 @@ srsly:
   - 2.*
 starlette:
   - 0.19.*
+sqlite:
+  - 3.38.2   # [s390x]
```

Comment:
Do we want this to be exact version ? OR 3.* or 3.38.* is better ?

Reply:
https://github.com/open-ce/open-ce/blob/open-ce-r1.5/envs/conda_build_config.yaml#L276

Reply:
I think we can relax it a bit by making 3.38.*.